github/compliance/report.py

```diff
@@ -431,7 +430,7 @@ def component_resource_label(issue: github3.issues.Issue) -> str:
     for result_group in result_groups:
         resource_label = github.compliance.issue.resource_digest_label(
             component=result_group.component,
-            resource=result_group.resource_name,
+            artifact=result_group.name,
```

Comment:
Same here, I would rather expect the newly introduced `<ScanResultGroup>.artifact()`.

Reply:
fixed in next commit